script_test.go

```diff
@@ -1850,6 +1850,82 @@ func TestReadReturnsErrorGivenReadErrorOnPipe(t *testing.T) {
 	}
 }
 
+func TestEncodeBase64(t *testing.T) {
+	t.Parallel()
+	tcs := []struct {
+		name string
+		s    string
+		want string
+	}{
+		{
+			name: "empty string",
+			s:    "",
+			want: "\n",
+		},
+		{
+			name: "single line string",
+			s:    "hello world",
+			want: "aGVsbG8gd29ybGQ=\n",
+		},
+		{
+			name: "multi line string",
```

Comment:
What do you think about adding some bytes cases? After all, it seems like it's exactly non-string data that people would want to encode to `base64`.

Reply:
@bitfield what would these tests look like? do you want something like `str([]byte("hello world"))`?

Reply:
What I mean is, suppose you had some arbitrary `byte` data (not necessarily a UTF-8-encoded string), and you ran it through a pipe with `EncodeBase64`. That should produce more or less the same string as piping the same bytes through a tool like `base64`, shouldn't it?

Reply:
@bitfield got it! i added a round trip test with a byte reader - let me know if this along the lines of what you were thinking